optuna_dashboard/ts/action.ts

```diff
@@ -582,6 +583,27 @@ export const actionCreator = () => {
         console.log(err)
       })
   }
+
+  const updatePreference = (
+    study_id: number,
+    best_trials: number[],
+    worst_trials: number[]
+  ) => {
+    reportPreferenceAPI(study_id, best_trials, worst_trials)
+      .then(() => {
+        setTimeout(() => {
+          updateStudyDetail(study_id)
+        }, 1000)
```

Comment:
I think you should not call `updateStudyDetail` after this logic since the study will be updated by API polling.